lib/preprocessors/glimmer.js

```diff
@@ -105,7 +105,8 @@ function mapRange(messages, filename) {
 
   return flattened.map((message) => {
     const line = lines[message.line - 1];
-    const token = line.slice(message.column - 1, message.endColumn - 1);
+    // `line` can be undefined here sometimes. I suspect maybe `message.line` is incorrect above.
```

Comment:
I wonder if this is due to file-wide messages, or maybe there is another property to use?
Anywho, happy to see this more resilient

Reply:
FWIW line numbers are way off for most of my error messages (usually 0:0). The fixes in this branch only get things running.

Reply:
turns out this is a legit scenario, because folks can cause parse errors, which occur on the whole file and don't have a line/column/etc